package.json

```diff
@@ -30,12 +30,12 @@
     "coverage": "c8 report --reporter=text-lcov > coverage.lcov && c8 report --reporter=html",
     "codecov": "codecov",
     "precodecov": "npm run coverage",
-    "build": "rollup -c && npm run tsd",
+    "build": "rollup -c",
     "prepare": "npm run build",
     "dev": "rollup -cw",
     "pretest": "npm run build",
     "posttest": "agadoo internal/index.mjs",
-    "prepublishOnly": "npm run lint && PUBLISH=true npm test",
+    "prepublishOnly": "npm run lint && npm run tsd && PUBLISH=true npm test",
```

Comment:
I realize it's completely trivial and unrelated to anything else going on here, but I can definitely appreciate moving generating the types to prepublish. They are an irritatingly large fraction of the time taken by `npm run build`.